docs/content/developers/building-contributing.md

```diff
@@ -2,7 +2,7 @@
 
 ## Testing latest commits
 
-You can download the latest artifacts from the master branch from [link](https://nightly.link/drud/ddev/workflows/master-build/master). On macOS and Linux you can `brew unlink ddev && brew install drud/ddev/ddev --HEAD --fetch-HEAD` to get the latest commit of ddev, even if it's not yet in a release.
+You can download the latest artifacts from the master branch from [link](https://nightly.link/drud/ddev/workflows/master-build/master). On macOS and Linux you can `brew unlink ddev && brew install drud/ddev/ddev --HEAD --fetch-HEAD` to get the latest commit of ddev, even if it's not yet in a release. If you are using WSL2, you will also need to install build tools with the following command: `sudo apt install build-essential gcc`.
```

Comment:
```suggestion
You can download the latest artifacts from the master branch from [link](https://nightly.link/drud/ddev/workflows/master-build/master). On macOS and Linux you can `brew unlink ddev && brew install drud/ddev/ddev --HEAD --fetch-HEAD` to get the latest commit of ddev, even if it's not yet in a release. If you are using linux/WSL2, you will also likely need to install build-essential with the following command: `sudo apt install -y build-essential`.
```

Reply:
I wasn't sure if Linux also required it. I assumed it was because WSL is a stripped-down.
I feel like `gcc` should be there because it does no harm and the error message specifically asks for it. But I am happy.

Reply:
build-essential includes gcc/g++ unless I'm mistaken.  The WSL2 Ubuntu is not necessarily more stripped down than any other Ubuntu install.